beta-src/src/components/svgr-components/ActionIconSelected.js

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+
+const SvgActionIconSelected = ({iconState}) => (
+  <svg
+    width={42}
+    height={42}
+    viewBox="6 3 42 42"
+    fill="none"
```

Comment:
i think this is default you can probably remove it? check and see.

Reply:
So this icon was actually one of the ones I mention in our previous meeting about the alignment being off center! These dimensions/viewbox are ones that I set so that the image is aligned with the svg (at least to the best of my ability lol). I'll remove "fill" though!